include/ignition/sensors/AirPressureSensor.hh

```diff
@@ -78,6 +78,10 @@ namespace ignition
       /// \return Verical reference position in meters
       public: double ReferenceAltitude() const;
 
+      /// \brief Check if there are any subscribers
+      /// \return True if there are subscribers, false otherwise
+      public: bool HasConnections() const;
```

Comment:
I assume you didn't make this a virtual function to avoid breaking ABI. How about adding a TODO to make it virtual on Garden?

Reply:
yep, added todo note in [bc0dfca](https://github.com/gazebosim/gz-sensors/pull/222/commits/bc0dfcada170fdf61f0de4aea0861ca9f69f71a5)